views/guests/donate.mote

```diff
@@ -5,7 +5,7 @@
 </div>
 
 <div class="container">
-  <div id = "donation-container" class = "row">
+  <div id = "donation" class = "row">
```

Comment:
A tiny details, inside templates I used to type attributes and values without spaces at the sides of the `=`, like:
`id="donation", instead of `id = "donation"`, I think it's more usual. And also, to have consistency in our templates. :-)